qiskit/primitives/containers/estimator_pub.py

```diff
@@ -101,48 +101,62 @@ def precision(self) -> float | None:
         return self._precision
 
     @classmethod
-    def coerce(cls, pub: EstimatorPubLike, precision: float | None = None) -> EstimatorPub:
+    def coerce(
+        cls, pub: EstimatorPubLike, precision: float | None = None, validate: bool = True
+    ) -> EstimatorPub:
         """Coerce :class:`~.EstimatorPubLike` into :class:`~.EstimatorPub`.
 
         Args:
             pub: A compatible object for coercion.
             precision: an optional default precision to use if not
                        already specified by the pub-like object.
+            validate: Whether to validate the returned pub.
```

Comment:
```suggestion
            validate: Whether to validate the pub before returning.
```
Technically its not returned if it fails validation